### DISCUSSION TITLE
Read Receipts

### DISCUSSION THREAD

**Discussion by trikotret** [simple-login/app#1128] [Feature Requests]
I like to suggest see message status of the emails we send using an alias. Have an option of getting notified that email was read. Since we are using alias emails and not real emails, I'm aways paranoid that the recipient won't get my email. This way if we have an option on SL dashboard to check status of read email.

**Comment by nguyenkims** [comment]
@trikotret when an email can't be delivered to a contact (i.e. bounced), SimpleLogin will send you a notification. If there's no notification, that means the email has been received successfully by the contact mail service. Please note that doesn't mean the email has been read or opened by the recipient, the only way to know if an email is opened or read is to put trackers into the email body which SimpleLogin is obviously against :).
